integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLargeCapacityDomainsClusters.java

```diff
@@ -0,0 +1,696 @@
+
+  private static int clusterReplicaCount = 2;
+  private static final int t3ChannelPort = getNextFreePort();
+  private static final String wlSecretName = "weblogic-credentials";
+  private static String wlsUserName = ADMIN_USERNAME_DEFAULT;
+  private static String wlsPassword = ADMIN_PASSWORD_DEFAULT;
+
+  private static String adminSvcExtHost = null;
+  private static String clusterRouteHost = null;
+  private static Path clusterViewAppPath;
+  private static LoggingFacade logger;
+  private static final int managedServerPort = 7100;
+
+  /**
+   * Assigns unique namespaces for operator and domains. Installs operator. Creates a WebLogic domain.
+   *
+   * @param namespaces injected by JUnit
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(50) List<String> namespaces) {
+    logger = getLogger();
+    logger.info("Assign a unique namespace for operator");
+    opNamespace = namespaces.get(0);
+    logger.info("Assign a unique namespaces for WebLogic domains");
+    domainNamespaces = namespaces.subList(1, numOfDomains + 1);
+    domainNamespace = namespaces.get(numOfDomains + 1);
+
+    // install operator and verify its running in ready state
+    installAndVerifyOperator(opNamespace, namespaces.subList(1, 50).toArray(new String[0]));
+
+    // build the clusterview application
+    Path targetDir = Paths.get(WORK_DIR,
+        ItLargeCapacityDomainsClusters.class.getName() + "/clusterviewapp");
+    Path distDir = BuildApplication.buildApplication(Paths.get(APP_DIR, "clusterview"), null, null,
+        "dist", domainNamespace, targetDir);
+    assertTrue(Paths.get(distDir.toString(),
+        "clusterview.war").toFile().exists(),
+        "Application archive is not available");
+    clusterViewAppPath = Paths.get(distDir.toString(), "clusterview.war");
+
+    createDomain(domainNamespace, domainUid);
+    // verify the admin server service and pod created
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace);
+    // verify managed server services and pods are created
+    for (int i = 1; i <= clusterReplicaCount; i++) {
+      logger.info("Checking managed server service and pod {0} is created in namespace {1}",
+          cluster1ManagedServerPodNamePrefix + i, domainNamespace);
+      checkPodReadyAndServiceExists(cluster1ManagedServerPodNamePrefix + i, domainUid, domainNamespace);
+    }
+    List<String> managedServerNames = new ArrayList<>();
+    for (int i = 1; i <= clusterReplicaCount; i++) {
+      managedServerNames.add(cluster1ManagedServerNameBase + i);
+    }
+    deployAndVerifyMemberHealth(adminServerPodName, managedServerNames, domainNamespace);
+
+  }
+
+  /**
+   * Test brings up new domains and verifies it can successfully start by doing the following.
+   * 
+   * a. Creates new WebLogic domains using offline WLST in persistent volume. 
+   * b. Creates domain resource and deploys in Kubernetes cluster. 
+   * c. Verifies the servers in the new WebLogic domain comes up.
```

Comment:
Verifies the servers in the new WebLogic domain comes up. -> Verifies the servers in the new WebLogic domain come up.